src/app/modules/time-clock/services/entry.service.ts

```diff
@@ -23,9 +23,9 @@ export class EntryService {
     return this.http.get(`${this.baseUrl}/running`);
   }
 
-  loadEntries(month): Observable<any> {
+  loadEntries(year, month): Observable<any> {
```

Comment:
Not critical. But I think it would be good to keep consistency across function calls, because
Here in load entries we have as parameteres `year, month`
and in constructor of entry actions(and subsequent calls), we have parameters `month, year`

I consider it would be good to keep same order of parameters.